system/t04_mirror/create.py

```diff
@@ -58,9 +58,7 @@ class CreateMirror6Test(BaseTest):
     """
     create mirror: missing release
     """
-    expectedCode = 1
```

Comment:
I don’t get why this line was removed.

Reply:
To answer myself: since this PR is trying to implement support for repositories missing Release, failure should not be expected...